index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="main">
+    <h1>Doctor Norman Borlaug</h1>
+    <div id="img-div">
+        <img  id="image" src="https://media.istockphoto.com/photos/mature-mixed-race-business-man-picture-id1059661424?k=20&m=1059661424&s=612x612&w=0&h=CLL4tto10GPo1gtMR9c-kPmf8VkvodjvTyqvtEuTLtg=">
+        <div id="img-caption"> Senior Sorfware Developer</div>
+    </div>
+    <div id="tribute-info">It is the page built with microverse peer!</div>
```

Comment:
I think it would be better to use semantic tags. You could replace the **div** tag with a **footer** tag.